naff/client/client.py

```diff
@@ -33,7 +33,55 @@
 
 import naff.api.events as events
 import naff.client.const as constants
-from naff.api.events import RawGatewayEvent, MessageCreate
+from naff.api.events import (
```

Comment:
We are already importing the events namespace on line 34, let's just use that, and refer to events as `events.Whatever`, the same way we do when dispatching them.